templates/mod/ssl.conf.erb

```diff
@@ -1,8 +1,16 @@
 <IfModule mod_ssl.c>
   SSLRandomSeed startup builtin
-  SSLRandomSeed startup file:/dev/urandom 512
   SSLRandomSeed connect builtin
+<% if @ssl_random_seeds and ! @ssl_random_seeds.empty? -%>
```

Comment:
Not necessarily, for instance - I set urandom to 1024 bits on my maschinen, removing the default 512 value.